panel-over ons.html

```diff
@@ -57,47 +66,55 @@
                         <li><a href="#contact">CONTACT</a></li>
                     </ul>
                     <h2>Over ons</h2>
-                    <div class="form-group red-border">
-                        <label for="text-form">Jouw text</label>
-                        <textarea class="form-control z-depth-1" id="text-form" placeholder="Typ wat in..." rows="10"></textarea>
-                        <button type="button" class="btn btn-outline-danger" id="button-left">+ Voeg een foto toe</button>
-                        <button type="button" class="btn btn-warning btn-primary" id="button-right">Opslaan</button>
-                    </div>
+                    <form method="post">
+                        <textarea id="summernote" name="editordata"></textarea>
+                    </form>
+                    <!-- <script>
+                        $(document).ready(function() {
+                            $('#summernote').summernote({
+                                height: 400,
+                                minHeight: 250,
+                                maxHeight: 1000,
+                                width: 1000,
+                                focus: true
+                            });
+                        });
+                    </script> -->
+                    <button type="button" class="btn btn-warning btn-primary" id="button-center">Opslaan</button>
+                    <button type="button" class="btn btn-dark" id="button-corner-right">Uitloggen</button>
                 </div>
-                <button type="button" class="btn btn-dark" id="button-corner-right">Uitloggen</button>
             </div>
         </div>
-    </div>
 
 
 
-    <!--footer-->
+        <!--footer-->
 
-    <footer class="section-footer">
+        <footer class="section-footer">
 
-        <div class="footer-container">
-            <div class="footer-info">
-                <h2>Bezoekadres:</h2>
-                <p> Markendaalseweg 75 <br> 4811 KB Breda <br> Nederland </p>
-            </div>
-            <div class="footer-post">
-                <h2>Correspondentieadres:</h2>
-                <p> Randoord 23 <br> 4824 LG Breda <br> Nederland
-                </p>
-            </div>
-            <div class="footer-free-space"></div>
-            <div class="footer-divider">
-                <div class="footer-terms-and-conditions">
-                    <h2><a href="privacy-policy.html">Algemene Voorwaarden</a></h2>
-                    @COPYRIGHT SHOWKORPS CONCENTO
+            <div class="footer-container">

```

Comment:
so those divs does not have names even thou from what I remember it was a good practice. 
Maybe if you don't want to remember to give it name because most of the time name will be the same as css class, try to search for extension to VSCode that will help you with that. I am sure it exists!

Reply:
`name` attribute is not supported by `div` tags. Only `class` or `id` to identify individual elements.

Reply:
I've created a new div for "copyright..". I saw a long time ago some code with a text without any div tag so I thought it's allowed.